doc/schema-management.md

```diff
@@ -36,13 +36,28 @@ order them in the correct order for applying to the database.
 
 ## Creating a Migration
 
-Whenever the Haskell schema definition in `Cardano.Db.Schema` is updated, a schema migration can
-be generated using the command:
+Whenever the Haskell schema definition in `Cardano.Db.Schema` is updated, a schema migration will need to be migrated.
+When migrating, `db-sync` caches some of the schema files when it is built so this can be point of confusion.
+
+Firstly you need to run current existing migrations by:
+```
+export PGPASSFILE=config/pgpass-mainnet
+cabal run cardano-db-tool -- run-migrations --mdir schema/
+```
+
+Once this has completed it's good practice to rebuild `cardano-db-sync` due to how it caches schema files when built, this can be done using the following documentation [Build and Install](./installing.md#build-and-install)
```

Comment:
I think this should be moved after the `create-migration`, because this is what invalidates the schema cache.